frontend/src/modules/Environments/views/EnvironmentCreateForm.js

```diff
@@ -395,7 +396,7 @@ const EnvironmentCreateForm = (props) => {
                             <Typography color="textPrimary" variant="subtitle2">
                               <CopyToClipboard
                                 onCopy={() => copyNotification()}
-                                text={`aws cloudformation --region REGION create-stack --stack-name DataAllCustomCDKExecPolicyStack --template-body file://cdkExecPolicy.yaml --parameters ParameterKey=EnvironmentResourcePrefix,ParameterValue=dataall --capabilities CAPABILITY_NAMED_IAM && aws cloudformation wait stack-create-complete --stack-name DataAllCustomCDKExecPolicyStack --region REGION && cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
+                                text={`aws cloudformation --region REGION create-stack --stack-name DataAllCustomCDKExecPolicyStack --template-body file://cdkExecPolicy.yaml --parameters ParameterKey=EnvironmentResourcePrefix,ParameterValue=dataall ParameterKey=NotebooksEnabled,ParameterValue=false ParameterKey=MLStudioEnabled,ParameterValue=false ParameterKey=PipelinesEnabled,ParameterValue=false ParameterKey=DashboardsEnabled,ParameterValue=false --capabilities CAPABILITY_NAMED_IAM && aws cloudformation wait stack-create-complete --stack-name DataAllCustomCDKExecPolicyStack --region REGION && cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
```

Comment:
I think maybe we leave `--parameters` empty here and in the CloudFormation template default to true 

So will be the same command as before and by default will give all permissions. Then can have documentation that discusses how to restrict further if needed